lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -1080,6 +1080,11 @@ def install_puppetlabs_dev_repo ( host, package_name, build_version,
           dev_builds_url = protocol + hostname
           dev_builds_url = opts[:dev_builds_url] if variant =~ /^(fedora|el|centos)$/
 
+          if variant =~ /^ubuntu$/ && version.split('.').first.to_i >= 18
```

Comment:
This is now redundant, since we immediately call `install_repo_configs` -> `install_repo_configs_from_url`.

Reply:
I still really dislike the redundancy here. And I think a spec test covering this would be useful (at the level of: this function ensures that we allow unsigned packages on Ubuntu 18.04).